### ISSUE TITLE
cramped vehicles

### ISSUE THREAD

**Issue by chefoxara** [CleverRaven/Cataclysm-DDA#74783]
### Is your feature request related to a problem? Please describe.

i cannot walk inside of a empty RV with a military backpack (33L) which isnt even much, ive walked around in a bus with a military backpack which was full of stuff and i could fit in pretty easily, of course i had to take it off so i could sit down but yeah. edit: for clarification im talking about me irl, not the game, ingame im p sure you cant walk around with a mil backpack in a bus too
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/79454795/bd862bf7-a104-463f-9727-3a03aab3e20f)
looking at the average RV i think i should be able to fit in, with problems while turning around ofc (cramped space effect to represent this)

### Solution you would like.

some sort of a calculation or a system that would calculate the tile inside of the vehicle you are on and the vehicle tiles around you and increase the volume so you can fit in if theres not too much stuff around you (i should be able to walk with little issue in a RV but shouldn't be able to sit down with a backpack obviously)

### Describe alternatives you have considered.

or a vehicle part you can build and install onto vehicle tiles to increase their space but in exchange decrease their health/make them get more damage from crashes and impacts?

**Comment by VimbaTheFish** [comment]
Can confirm that it works a bit odd right now
Tested it a bit with my regular survivor with little overweight and 178 cm
I tried to move inside firetruck with equipment and backpack, and that's what I got:

1. If I move in empty tile with not damaged frame, roof and aisle, I squeese
2. If I move in empty tile with XX damaged aisle and undamaged frame, I move freely
3. If I move in empty tile with XX damaged FRAME and undamaged aisle, I move freely

Something tells me, that aisle and frame volumes are taken in free space count calculations, though I dunno is it intended or not
Perhaps the problem is counting aisle at all? It should make moving more doable, but now it crampes space

Proofs:  
1:
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/22179405/17026e20-3d07-48b4-9d41-412c95c297f5)
2:
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/22179405/5305d6e6-69f1-4c75-bf4f-d567b00c9234)
3:
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/22179405/d7b6ca77-c5fa-4e29-b91f-82ff327380ce)

Also, I cannot seat comfortable in reclining seat. Event without clothes and backpacks. Perhaps it's overweight, but still, seat it reclining for a reason, and I have only first step of overweight, so I also dunno is it intended or not

**Comment by foxymacko-real** [comment]
yeah its very weird rn, but they made this change only like a month ago or smth so i dont expect it to be perfect from the start, i just hope they fix it and smooothhh it out so its good and makes sense for both gameplay and realism

**Comment by NerosShadow** [comment]
Can likewise confirm that moving in and out of vehicles in full military kit, ie. Clothing, armor, vest w/ equipment, and rifle is debilitating however this is extreme. Military Vehicles are notoriously cramped, but anything with "Bucket Seats" is much less so.
Is the crampedness supposed to effect your driving as well? I am fumbling controls with vehicles lvl 6.
